tide_webform.module

```diff
@@ -322,6 +322,13 @@ function tide_webform_webform_submission_presave(WebformSubmission $webform_subm
         $webform_submission->setData($submission_data);
       }
     }
+    // Server side validation from API submission.
+    if ($element['#required'] == TRUE) {
```

Comment:
After reading the contrib issue it seems like it hasnt been progressed as there is no clear way to enforce a mandatory field on a multi-step form. 

Can we add a condition here that checks the form isn't multi-step?

Reply:
done, Added check for multi step, if it multi step it will exit from the element loop and will not the validation. We also currently do not support multi step form.

Reply:
Understood - just want to ensure we don't break anything on the many drupal sites that we aren't directly managing.